### PR TITLE
PENDING UPDATE:Add cycle example

### PR DESCRIPTION
**Pull request by chrisprice**
DON'T MERGE UNTIL CYCLE CHANGES HAVE BEEN RELEASED

Adds cycle example to the website based on @godds example.

### CODE REVIEW

File: series/_posts/2010-10-7-cycle.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  
+  var parseDate = d3.time.format('%Y%m%d').parse;
+  data.forEach(function(d) {
+      d.date = parseDate(d.date);
+  });
+
+  var dayScale = d3.scale.linear()
```
Comment (ColinEberhardt) [inline series/_posts/2010-10-7-cycle.md]: I think it makes more sense to encode this as an ordinal axis:

        var dayScale = d3.scale.ordinal()
          .rangePoints([0, plotArea.layout('width')], 1.0)
          .domain(['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday']);

It makes the scale padding a bit more explicit

File: series/_posts/2010-10-7-cycle.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      .subScale(subScale)
+      .subSeries(multiSeries);
+
+  container.append('g')
+      .datum(data)
+      .call(cycle);
+---
+
+The cycle series component allows the creation of cycle plots which can help identify cyclical patterns nested in time series data. The following example shows some arbitrary sales data grouped by weekday:
+
```
Comment (ColinEberhardt) [inline series/_posts/2010-10-7-cycle.md]: I think this example could do with a little more explanation, the use of two different scales is notable! The `xScale` is the 'outer' scale, with the cycle series `xValue` mapping the data onto this scale, while the `subScale` is the inner scale, with the `xValue` of the line series corresponding to this scale.

I'd also mention the axis, that it is the same as the d3 axis, but with the addition of a baseline function which maps the array of data to a y-location (in this example)

**Comment by ColinEberhardt** [comment]
This is a nice and simple example, I just think it needs a few more words to describe how it works :-)

File: series/_posts/2010-10-7-cycle.md (hunk 1 above)
Comment (chrisprice) [inline series/_posts/2010-10-7-cycle.md]: Changed, not sure it's possible to work the weekday strings in though -
```
var dayScale = d3.scale.ordinal()
      .domain([0, 1, 2, 3, 4, 5, 6])
      .rangePoints([0, width], 1.0);
```

**Comment by chrisprice** [comment]
http://chrisprice.io/d3-financial-components/components.html#cycle

**Comment by ColinEberhardt** [comment]
:+1:

**Pull request closed without merging**